src/pantry.js

```diff
@@ -1,25 +1,38 @@
 class Pantry {
-	constructor(givenUser) {
-		this.data = givenUser.pantry;
+  constructor(givenUser) {
+    this.data = givenUser.pantry;
+  }
+
+  checkPantry(clickedRecipe, dataset1, dataset2) {
+    const foundRecipe = dataset1.find(recipe => {
+      return recipe.id === clickedRecipe.id;
```

Comment:
this could be a one liner

Reply:
you're right! updated that method, tired brain made some redundant code.. it's being passed the ingredient dataset now, b/c it needs to parse out the name of the ingredients from their id's, the names are only held in the ingredients dataset